test/parse_request.cc

```diff
@@ -138,6 +138,7 @@ constexpr float kDefaultTruck_CountryCrossingPenalty = 0.0f;   // Seconds
 constexpr float kDefaultTruck_LowClassPenalty = 30.0f;         // Seconds
 constexpr float kDefaultTruck_TruckWeight = 21.77f;            // Metric Tons (48,000 lbs)
 constexpr float kDefaultTruck_TruckAxleLoad = 9.07f;           // Metric Tons (20,000 lbs)
+constexpr float kDefaultTruck_TruckAxles = 3.f;                // Count
```

Comment:
the rest of the defaults are for semi trucks which have 5 axles right? 3 for the tractor and 2 more for the trailer

Reply:
There are a bunch of classes/configurations and trailer types for hauling. Here is a decent overview: https://www.fhwa.dot.gov/policyinformation/tmguide/tmg_2013/vehicle-types.cfm

As for the 2, 3, 5 I added... those were just random.

Reply:
oh jeez im sorry i thought i was looking at the truck costing defaults. my eyes saw the names of the constants and i didnt even look at the file name. i thought i was looking here: https://github.com/valhalla/valhalla/blob/e8220b7e82411b310eb275b01f4774f5bfdb2409/src/sif/truckcost.cc#L50-L54